### PR TITLE
Add rapidsai (cudf and cuml)

### PR DESCRIPTION
**Pull request by jperez999**

### CODE REVIEW

**Comment by jperez999** [comment]
To run jupyterlabs in the container and have access to the gpu in the notebooks I had to use the following docker run command:
```
docker run --runtime nvidia --rm -it -e HOME=/tmp -w=/tmp -v /tmp/empty_dir:/usr/local/cuda/lib64/stubs:ro -p 8888:8888 -p 8787:8787 -p 8786:8786 kaggle/python-gpu-build
```

**Comment by Philmod** [comment]
The CI tests fail with: 
```
======================================================================
ERROR: test_cudf (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_cudf
Traceback (most recent call last):
  File "/opt/conda/lib/python3.6/unittest/loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "/opt/conda/lib/python3.6/unittest/loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "/input/tests/test_cudf.py", line 2, in <module>
    import cudf as gd
ModuleNotFoundError: No module named 'cudf'


======================================================================
ERROR: test_cuml (unittest.loader._FailedTest)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_cuml
Traceback (most recent call last):
  File "/opt/conda/lib/python3.6/unittest/loader.py", line 428, in _find_test_path
    module = self._get_module_from_name(name)
  File "/opt/conda/lib/python3.6/unittest/loader.py", line 369, in _get_module_from_name
    __import__(name)
  File "/input/tests/test_cuml.py", line 4, in <module>
    from cuml import LinearRegression as cuLinearRegression
ModuleNotFoundError: No module named 'cuml'
```

**Comment by jperez999** [comment]
Moved GPU specific imports into the GPU tests themselves to resolve import failures on cpu container. We also cannot incorporate newer version of rapids until pandas is updated >=0.24.2, <0.25.

File: tests/test_cuml.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+import unittest
+
+from sklearn import datasets
+from cuml import LinearRegression as cuLinearRegression
```
Comment (Philmod) [inline tests/test_cuml.py]: You may have forgotten to remove these?

**Comment by Philmod** [comment]
> We also cannot incorporate newer version of rapids until pandas is updated >=0.24.2, <0.25.

Is this blocking this PR?

File: tests/test_cuml.py (hunk 1 above)
Comment (jperez999) [reply tests/test_cuml.py]: Your right I will update that.

**Comment by jperez999** [comment]
No rapids version 0.8 works fine. However, the most recent released version is 0.9 with 0.10 coming soon. Those versions of rapids will be blocked by that pandas requirement (>=0.24.2, <0.25).

**Comment by Philmod** [comment]
Thank you for your contribution!

**Comment by Philmod** [comment]
Unfortunately, we are moving to terraform 2 and it looks like the library is not supported. https://github.com/Kaggle/docker-python/pull/573/files

**Pull request merged**